README.md

```diff
@@ -51,6 +51,7 @@ A curated list of Minio community projects inspired by [awesome-go](https://gith
 * [Trustvox](http://site.trustvox.com.br) - Trustvox is an integrator of ecommerce platforms, ensures 100% valid product reviews, allowing only real buyers to post reviews along with a continuous audit process. The platform team chose Minio to create its own S3 service to avoid infrastructure vendor lock-in, obtaining transparent compatibility with Google Cloud Storage and previous Amazon S3 infrastructures. We tested other options before, and only Minio showed the stable and enterprise-grade results we were looking for.
 * [www.houbank.com](https://www.houbank.com/home/) - Houbank uses Minio as a simple, reliable and compatible self-host AWS S3 alternative. Houbank uses Minio to upload user files and exchange documents in internal applications/jobs.
 * [Rocket.Chat](https://rocket.chat/) - Rocket.Chat is a completely FOSS alternative to Slack. In Rocket.Chat Minio can be used for file uploads — this applies to channels, groups and privately between one user and another.
+* [Plutify](http://plutify.com/) - Plutify is a scalable Cloud Storage Space for store and serve client files beside variety of service like video sharing, file selling, direct link and ... which uses Minio Object Storage in order to keep clients uploaded files.
```

Comment:
@ehsanagh not clear what ` and ...` means here, could you please clarify

Reply:
are you planning to fix it @ehsanagh ?